eisbuk-admin/src/components/atoms/AttendanceCard/AttendanceCard.tsx

```diff
@@ -36,28 +66,86 @@ const AttendanceCard: React.FC<Props> = ({
   const endTime = luxonEnd.toISOTime().substring(0, 5);
 
   const timeString = `${startTime} - ${endTime}`;
+
   return (
-    <div>
-      <div data-testid="time-string">{timeString}</div>
-      <div>{categories}</div>
-      {userBookings.map((user) => {
-        const isAbsent = absentees?.includes(user.customer_id) || false;
-        return (
-          <div key={user.customer_id}>
-            <div>{user.name}</div>
-            <button
-              type="button"
-              onClick={() =>
-                dispatch(markAttendance(user.customer_id, isAbsent))
+    <Container maxWidth="sm">
+      <List className={classes.root}>
+        <div key={id + "-" + timeString} className={classes.slotWrapper}>
+          <ListItem className={classes.listHeader}>
+            <ListItemText
+              primary={
+                <span>
+                  {timeString} <b>({userBookings.length})</b>
+                </span>
               }
-            >
-              {isAbsent ? "👎" : "👍"}
-            </button>
-          </div>
-        );
-      })}
-    </div>
+              secondary={translateAndJoinTags(categories, type)}
+            />
+          </ListItem>
+          {userBookings.map((user) => {
+            const isAbsent = absentees?.includes(user.customer_id) || false;
+            const listItemClass = isAbsent ? classes.absent : "";
+
+            const absenteeButtons = (
+              <Button
+                variant="contained"
+                size="small"
+                color={isAbsent ? "primary" : "secondary"}
+                onClick={() =>
+                  dispatch(markAttendance(user.customer_id, isAbsent))
+                }
+                // disabled={hasLocalChange}
+              >
+                {isAbsent ? "👎" : "👍"}
+              </Button>
+            );
+
+            return (
+              <ListItem
+                key={`${id}-${user.customer_id}`}
+                className={listItemClass}
+              >
+                <ListItemAvatar>
+                  <EisbukAvatar {...user} />
+                </ListItemAvatar>
+                <ListItemText primary={user.name} />
+                <ListItemSecondaryAction>
+                  {absenteeButtons}
+                </ListItemSecondaryAction>
+              </ListItem>
+            );
```

Comment:
You literally need to copy this part of the code in to new fuctional component `UserAttendance`, TypeScript will tell you which imports/props you're missing. Add those props into local `interface Props` and consume the components in here like this
```
{userBookings.map(userBooking => <UserAttendance {...userBooking} />)}
```

Reply:
I know, that's why I titled the commit as unfinished; I wanted to leave `AttendanceCard` as it is until `UserAttendance` was finished (I just pushed the finished version)